### PR TITLE
New-267-added modal with progressbar

### PR DESCRIPTION
**Pull request by VaabelMaret**

### CODE REVIEW

**Review by IngaSokman — APPROVED** [review]

File: src/components/organisms/modals/FormProgressModal/FormProgressModal.tsx
Comment (KaarelKa) [inline src/components/organisms/modals/FormProgressModal/FormProgressModal.tsx:25]: **[non-blocker]**

Not necessary, but a less error-prone data structure here would be to combine "formData" and "steps" props here.
Basically sth like


`
steps = [
    {
        label: 'Step1',
        title: 'Title1',
        helperText: 'Helper1',
        content: <div>Content1>
    },
    ...
]
`

This way it will be very clear what step matches what content + helps us avoid cases, where the size of "steps" and "formData" arrays don't match.

**Review by KaarelKa — APPROVED** [review]

File: src/components/organisms/modals/FormProgressModal/FormProgressModal.tsx
Comment (VaabelMaret) [reply src/components/organisms/modals/FormProgressModal/FormProgressModal.tsx:25]: Good Idea

**Comment by KaarelKa** [comment]
@VaabelMaret You can merge this, after merge conflicts are resolved

**Pull request merged**
